extensions/frontend-tiles/src/integration-test/IntegrationTest.test.ts

```diff
@@ -0,0 +1,108 @@
+import { IModelApp } from "@itwin/core-frontend";
+import { expect } from "chai";
+import { TestContext } from "./TestContext";
+import { GraphicRepresentationStatus } from "../GraphicsProvider/GraphicRepresentationProvider";
+
+async function getExport(exportId: string, accessToken: string, urlPrefix?: string): Promise<any> {
```

Comment:
any? we dont have this type defined anywhere?

Reply:
extract the type out, mark it as internal, and re-use it here

Reply:
I was actually incorrect. We don't have this type defined anywhere in the frontend-tiles package. We have a very similar type defined in `GraphicRepresentationProvider.ts`, but that one is the response back from using the get-exports API, whereas here I am using the get-export (no s) API. That structure is slightly different, and as far as I can see, is not defined anywhere here.

Reply:
are we going to type this out?

Reply:
I don't believe it's necessary to type it out, as it's only used here for this test. If that's a sticking point, I can type it out maybe in `GraphicRepresentationProvider.ts` and export it here.